java/osgi/OsgiWrapper.java

```diff
@@ -138,6 +144,9 @@ public Integer call() throws Exception {
       if (analyzer.save(outputJar, true)) {
         return 0;
       }
+    } else {
+      List<String> err = analyzer.getErrors();
+      System.out.println("BND analyzer failed: " + err.toString());
```

Comment:
I added the ability to print errors from BND tools during dev and felt it may be useful to keep.